charts/cockroachdb/19.1.5/questions.yml

```diff
@@ -0,0 +1,10 @@
+categories:
+- Config
+questions:
+- variable: Storage
+default:  "100Gi"
```

Comment:
The tabs indentation of the following variables are incorrect, can u please help to update it, thanks.
![image](https://user-images.githubusercontent.com/4569037/68849999-20d52e80-070e-11ea-9d0e-2709dd02f344.png)

Reply:
It seems storage is required of the CockroachDB app, so `StorageClass` should be a required setting.
you can refer https://github.com/rancher/charts/blob/master/charts/docker-registry/v1.8.1/questions.yml#L141 as an example.

Reply:
Technically, if a customer wants to manually provision PVs they do not need to set a storageclass, which is why we don't set it as required.

Reply:
I did however reformat the yaml file